ci_framework/roles/repo_setup/tasks/install.yml

```diff
@@ -26,7 +26,7 @@
   ansible.builtin.pip:
     virtualenv: "{{ cifmw_repo_setup_basedir }}/venv"
     requirements: "{{ cifmw_repo_setup_basedir }}/tmp/repo-setup/requirements.txt"
-    virtualenv_command: "python3 -m venv"
```

Comment:
with pip 23.3 from /usr/lib/python3/dist-packages/pip (python 3.11)
"python3 -m venv" results in a venv with no packages installed.

so on fedora 39 this venv does not have setuptools installed 

so cmd: "{{ cifmw_repo_setup_basedir }}/venv/bin/python setup.py install" below fails.

i am currently hitting this on a downstream system that currently cant boot with rhel 9 or centos 9 stream. it looks like a issue with either the relevant kernel or the relvent internal images so I'm currently using fedora 39 as a workaround.

Reply:
oh i didnt know gibi had filed that yes ill mention that here.
this should also work with centos 9 stream or rhel 9.

the delta seams to be that on newer systems venv just does not include setuptools by default so that is why we only see this failing there.